languages/javascript/exercises/concept/array-destructuring/array-destructuring.js

```diff
@@ -0,0 +1,21 @@
+export function getFirstCard(deck) {
+  throw new Error('Implement the getFirstCard function')
+}
+
+export function getSecondCard(deck) {
+  throw new Error('Implement the getSecondCard function')
+}
+
+export function swapTopTwoCards(deck) {
+  throw new Error('Implement the swapTopTwoCards function')
+}
+
+export function discardTopCard(deck) {
+  throw new Error('Implement the discardTopCard function')
+}
+
+export function insertFaceCards(deck) {
+  const faceCards = ['jack', 'queen', 'king']
```

Comment:
Can you extract this to a _top level constant_ instead? It can be right above this function.